tox.ini

```diff
@@ -11,15 +11,28 @@ setenv =
 commands =
     py.test -vvv {posargs}
 
-[testenv:lint]
+[testenv:black]
+description = black checks [Mandatory]
+skip_install = true
 deps =
-    flake8
+    black==19.10b0
```

Comment:
What's the reason for pinning the deps here and in `testenv:flake8` ?

Reply:
I added a comment explaining this.